game/theater/controlpoint.py

```diff
@@ -839,11 +838,10 @@ def capture(self, game: Game, events: GameUpdateEvents, for_player: bool) -> Non
         # All the attached TGOs have either been depopulated or captured. Tell the UI to
         # update their state. Also update orientation and IADS state for specific tgos
         for tgo in self.connected_objectives:
-            if isinstance(tgo, IadsGroundObject) or isinstance(
-                tgo, VehicleGroupGroundObject
-            ):
-                if isinstance(tgo, IadsGroundObject):
-                    game.theater.iads_network.update_tgo(tgo)
+            is_vehicle_go = isinstance(tgo, VehicleGroupGroundObject)
+            if tgo.is_iads or is_vehicle_go:
+                if tgo.is_iads:
+                    game.theater.iads_network.update_tgo(tgo, events)
```

Comment:
Should still be doing the same, just a readability improvement...

Reply:
This is a behavior change because `ShipGroundObject` now counts. Maybe that was intentional, but your comment says otherwise (this is what I mean when I say "multi-task PRs are hard to review", I'm not sure what the _intent_ of either change is)

Reply:
You're right, but look at iadsrole.py, clearly shows that naval units act as EWR in the network so indeed it is not doing the same in the sense that it includes a fix for naval units...

Reply:
Ok I realize I need to rework this, because the iads update has to happen for all iads units connected to the CP, but comm towers and power stations shouldn't rotate as compared to a sam/ewr site...

Reply:
Resolving this as I have corrected this part, will become available in the next push...